react/features/transcription/components/TranscriptionLabel.native.js

```diff
@@ -0,0 +1,3 @@
+/**
+ * Created by ystamcheva on 12/9/17.
+ */
```

Comment:
I usually don't see headers/created stamps in jitsi-meet.

Reply:
Will remove, my IDE adds them. I need to find the right configuration to disable this.